content/blog/day-2-kubernetes-migrating-eks-nodegroups-with-zero-downtime/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Day 2 Kubernetes - Migrating EKS Node Groups with Zero Downtime"
+authors: ["mike-metral"]
+tags: ["Kubernetes","EKS", "Migrations", "Node Groups"]
+date: "2019-07-18"
+
+meta_image: "eks-migrate-nodegroups.svg"
+---
+
+Managed Kubernetes clusters greatly reduce the overhead required in
+administering Kubernetes. However, the cluster is only one of the
+components under management, as app lifecycles are self-driven tasks
+that vary by workloads.
+
+In Kubernetes, node groups are a useful mechanism of creating pools of resources that can enforce
+scheduling requirements. They also provide a utility for shifting
+workloads around during cluster management and updates.
+
+In this post, we'll see how to use Pulumi for Day 2 Kubernetes administration.
+We'll spin up a new EKS cluster with two node groups, and a given workload.
+Then we'll add an additional, updated node group, and migrate the workload
```

Comment:
I'd rephrase "add an additional" to "add one more" but I'm not sure what you mean by "updated". Will "add one more updated node group" work as a substitution here?

Reply:
Rephrased to: "Then we'll add one more node group with updated configuration..."